README.md

```diff
@@ -1 +1,23 @@
-# group13_532
\ No newline at end of file
+# Movie Dashboard
+
+## Description of the app
+
+Have you ever think about how many movies are produced in each country every year? how many of them categorized in each movie rating groups? what is the most populare genres among them? This data visualizatin app aims to give you this insight that what is going on in movie world in the countries?
```

Comment:
My only concern here is that this is pretty broad and non-specific. I think it would be more clear to explicitly mention Netflix because our data is only from Netflix! For example, in the proposal I wrote, "Netflix, one of the most popular streaming services available, is an ideal candidate for film studios to gather insights about global movie trends. We propose building a dashboard that visualizes the breadth of films that Netflix has released from 2009 to 2019. "